ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -3233,6 +3233,187 @@ spec:
+    -  5029: The Windows Firewall Service failed to initialize the driver. The service will continue to enforce the current policy.
+    -  5030: The Windows Firewall Service failed to start.
+    -  5032: Windows Firewall was unable to notify the user that it blocked an application from accepting incoming connections on the network.
+    -  5033 : The Windows Firewall Driver has started successfully.
+    -  5034 : The Windows Firewall Driver has been stopped.
+    -  5035 : The Windows Firewall Driver failed to start.
+    -  5037 : The Windows Firewall Driver detected critical runtime error. Terminating.
+    -  5058: Key file operation.
+    -  5059: Key migration operation.
+    The recommended state for this setting is: 'Success and Failure'.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the following UI path to include Success and Failure:
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Advanced Audit Policy Configuration\Audit Policies\System\Audit Other System Events'
+  query: |
+    SELECT 1 FROM mdm_bridge where mdm_command_input = "<SyncBody><Get><CmdID>1</CmdID><Item><Target><LocURI>./Device/Vendor/MSFT/Policy/Result/Audit/System_AuditOtherSystemEvents</LocURI></Target></Item></Get></SyncBody>" 
+    AND mdm_command_output = 3;
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_17.9.2
+  contributors: sharon-fdm
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Audit Security State Change' is set to include 'Success'
+  platforms: win10
+  platform: windows
+  description: |
+    This subcategory reports changes in security state of the system, such as when the security subsystem starts and stops. Events for this subcategory include:
+    -  4608: Windows is starting up.
+    -  4609: Windows is shutting down.
+    -  4616: The system time was changed.
+    -  4621: Administrator recovered system from CrashOnAuditFail. Users who are not
+    administrators will now be allowed to log on. Some audit-able activity might not have been recorded.
+    The recommended state for this setting is to include: 'Success'.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the following UI path to include Success:
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Advanced Audit Policy Configuration\Audit Policies\System\Audit Security State Change'
+  query: |
+    SELECT 1 FROM mdm_bridge where mdm_command_input = "<SyncBody><Get><CmdID>1</CmdID><Item><Target><LocURI>./Device/Vendor/MSFT/Policy/Result/Audit/System_AuditSecurityStateChange</LocURI></Target></Item></Get></SyncBody>" 
+    AND (mdm_command_output = 3 OR mdm_command_output = 1);
```

Comment:
Question on "mdm_command_output = 3", this check is going to include success and failure states right?

Reply:
Thanks for the answer. The query should only check for "mdm_command_output = 3" as the policy checks that both  'Success and Failure' are set.

Reply:
Hmmm 🤔 , I think you are wrong 🙂 
`Ensure 'Audit Security State Change' is set to include 'Success'`

Reply:
you are correct, I looked into the wrong policy that stated 'Success and Failure'